packages/webapp/tsconfig.json

```diff
@@ -17,13 +17,20 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "strictNullChecks": true
```

Comment:
@rebelchris I did not change this explicitly, was there some reverted commits from the main branch about tsconfig?